web/app/js/components/ResourceDetail.jsx

```diff
@@ -28,7 +28,7 @@ import { withContext } from './util/AppContext.jsx';
 // if there has been no traffic for some time, show a warning
 const showNoTrafficMsgDelayMs = 6000;
 // resource types supported when querying API for edge data
-const edgeDataAvailable = ["daemonset", "deployment", "job", "pod", "replicationcontroller", "statefulset"];
+const edgeDataAvailable = ["daemonset", "deployment", "job", "pod", "replicationcontroller", "statefulset", "replicaset", "cronjob"];
```

Comment:
This is a tiny nit, but in general in the JS code we have the resources alphabetized for readability - could you alphabetize these as well?